app/src/main/java/dev/shreyaspatil/covid19notify/utils/ThemeHelper.kt

```diff
@@ -5,17 +5,17 @@ import android.content.res.Configuration
 import androidx.appcompat.app.AppCompatDelegate
 
 object ThemeHelper {
-    private const val lightMode = "light"
-    private const val darkMode = "dark"
-    private const val batterySaverMode = "battery"
-    const val default = "default"
+    private const val LIGHT_MODE = 0
+    private const val DARK_MODE = 1
+    private const val BATTERY_SAVER_MODE = 2
+    const val DEFAULT = 3
```

Comment:
Why this isn't `private`?

Reply:
Because in application class we are using it.

Reply:
Yes, constants should be always public and static. I think it should be moved to `companion object` to access it from outside class.